Pod/Classes/MRGArchitectJSONLoader.m

```diff
@@ -155,7 +155,13 @@ - (NSDictionary *)loadEntriesWithClassName:(NSString *)className {
 }
 
 - (NSString *)pathForClassName:(NSString *)className suffix:(NSString *)suffix {
-    NSString *path = [[NSBundle mainBundle] pathForResource:[NSString stringWithFormat:@"%@%@", className, suffix ?: @""] ofType:@"json"];
```

Comment:
```objc-c
NSString *path = [[[NSBundle mainBundle] bundlePath] stringByAppendingPathComponent:[NSString stringWithFormat:@"%@%@.json", className, suffix ?: @""]];
```

^ ça tout simplement ça ne fonctionnait pas?

Reply:
Lors de la précédente PR, j'avais eu le commentaire suivant : 
> Was it so that if the class was in a pod's bundle instead of the app bundle it would still get it? 

Afin de m'assurer de préserver le support avec les _bundles pod_, j'ai opté pour cette solution.

Si ce n'est pas nécessaire, je peux simplement utiliser 
```
NSString *path = [[[NSBundle mainBundle] bundlePath] stringByAppendingPathComponent:[NSString stringWithFormat:@"%@%@.json", className, suffix ?: @""]];
``` 
😄